### PR TITLE
improve FAQ section of readme

### PR DESCRIPTION
**Pull request by remonh87**
Provided a better explanation of the undeliverable exception and also added a section about permissions

### CODE REVIEW

**Comment by remonh87** [comment]
Fixes #88 and #89

File: README.md [hunk 1]
```diff
@@ -173,3 +178,24 @@ RxJavaPlugins.setErrorHandler { throwable ->
...
+* ACCESS_FINE_LOCATION : this permission is needed because old Nexus devices need location services in order to provide reliable scan results
+* BLUETOOTH : allows apps to connect to a paired bluetooth device
+* BLUETOOTH_ADMIN: allows apps to discover and pair bluetooth devices
+
+These permissions are already added in the manifest of the this library and thus should automatically merge
+into the manifest of your app. It is not needed to add the permissions in your manifest.
+
+**iOS**
+
+For iOS it is required you add the following entries to the info.plist file of your app. It is not required to access Core BLuetooth without this. See [our example app](https://github.com/PhilipsHue/flutter_reactive_ble/blob/master/example/ios/Runner/Info.plist) on how to implement this. For more indepth details: [Blog post on iOS bluetooth permissions](https://medium.com/flawless-app-stories/handling-ios-13-bluetooth-permissions-26c6a8cbb816)
```
Comment (PieterAelse) [inline README.md:195]: Do you mean "allowed" instead of "required'?

File: README.md [hunk 2]
```diff
@@ -155,13 +155,18 @@ The Android OS maintains a table per device of the discovered service in cache.
...
 
-#### BLE undeliverable exception
+#### How to handle the BLE undeliverable exception
 
 On Android side we use the [RxAndroidBle](https://github.com/Polidea/RxAndroidBle) library of Polidea. After migration towards RxJava 2 some of the errors are not routed properly to their listeners and thus this will result in a BLE Undeliverable Exception. The root cause lies in the threading of the Android OS. As workaround RxJava has a hook where you can set the global errorhandler. For more info see [RxJava docs](https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling) .
 
-A default workaround implementation in the Flutter app (needs to be in the Java / Kotlin part e.g. mainactivity) would be:
+
+A default workaround implementation in the Flutter app (needs to be in the Java / Kotlin part e.g. mainactivity) is shown below. An example for Java is displayed on 
+the [Polidea ble library](https://github.com/Polidea/RxAndroidBle/tree/master/sample/src/main/java/com/polidea/rxandroidble2/sample)
```
Comment (werediver) [inline README.md:166]: I'd suggest to amend the last sentence like

```markdown
For an example (in Java) see Polidea RxAndroidBle [sample](https://...).
```

File: README.md [hunk 3]
```diff
@@ -155,13 +155,18 @@ The Android OS maintains a table per device of the discovered service in cache.
...
+
+BleException is coming from Polidea so make sure in your application to add the following depedency:
```
Comment (werediver) [inline README.md:168]: I'd suggest something like

```markdown
`BleException` is coming from Polidea RxAndroidBle, so make sure your application declares the following depedency:
```

File: README.md (hunk 1 above)
Comment (werediver) [reply README.md:195]: I guess, @PieterAelse means this sentence:

```markdown
It is not required to access Core BLuetooth without this.
```

Additionally, the "info.plist" should be "Info.plist" (capitalized), and I'd put a file name in double quotes.

**Comment by werediver** [comment]
CI fails with the following message:

```
00:29 +204: All tests passed!                                                                                                                                                                          
+ cd example/android
+ ./gradlew detekt
/var/folders/r7/d9twdq011sb8d3q1p8f39cdr0000gn/T/build_script_4_0_sv0n0a7z: line 3: ./gradlew: No such file or directory


Build failed :|
Step 5 script `Post-test script` exited with status code 1
```

File: README.md (hunk 1 above)
Comment (remonh87) [reply README.md:195]: good catch I mean indeed it is not allowed :)

File: example/android/gradlew [hunk 4]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```
Comment (werediver) [inline example/android/gradlew:5]: Is it really a normal practice to include these scripts as a part of a project?

**Pull request merged**